### PR TITLE
fixed-personal-profile

### PR DESCRIPTION
**Pull request by solomiyon**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by sharix4 — APPROVED** [review]

**Review by andrii52 — APPROVED** [review]

File: src/pages/userPage/personalData/AvatarAndProgressStatic.tsx [hunk 1]
```diff
@@ -206,16 +267,15 @@ const AvatarAndProgressStatic: React.FC<AvatarAndProgressStaticProps> = (
...
-                {dist.distinction.name.slice(0,54) + "..."} №{dist.number}
+                {dist.distinction.name.slice(0, 54) + "..."} №{dist.number}
               </h2>
             </Tooltip>
-          : <h2> 
-              {dist.distinction.name + "№" + dist.number} 
-            </h2>
-          }
+          ) : (
+            <h2>{dist.distinction.name + "№" + dist.number}</h2>
```
Comment (christykobyletska) [inline src/pages/userPage/personalData/AvatarAndProgressStatic.tsx:277]: two different styles for <h2></h2>

**Review by nicoolle — APPROVED** [review]

**Review by sharix4 — APPROVED** [review]

File: src/pages/userPage/personalData/AvatarAndProgressStatic.tsx [hunk 2]
```diff
@@ -206,16 +267,15 @@ const AvatarAndProgressStatic: React.FC<AvatarAndProgressStaticProps> = (
       </div>
       {UserDistinctions.map((dist) => (
         <div className="distinctions">
-          {(dist.distinction.name?.length > nameMaxLength) ?
+          {dist.distinction.name?.length > nameMaxLength ? (
             <Tooltip title={dist?.distinction.name}>
               <h2>
-                {dist.distinction.name.slice(0,54) + "..."} №{dist.number}
+                {dist.distinction.name.slice(0, 54) + "..."} №{dist.number}
```
Comment (christykobyletska) [inline src/pages/userPage/personalData/AvatarAndProgressStatic.tsx:273]: here there is one type of tabulation for h2, belove for h2 another

**Review by christykobyletska — CHANGES_REQUESTED** [review]

File: src/pages/userPage/personalData/AvatarAndProgressStatic.tsx (hunk 2 above)
Comment (solomiyon) [reply src/pages/userPage/personalData/AvatarAndProgressStatic.tsx:267]: fixed

File: src/pages/userPage/personalData/AvatarAndProgressStatic.tsx (hunk 1 above)
Comment (solomiyon) [reply src/pages/userPage/personalData/AvatarAndProgressStatic.tsx]: fixed

**Review by christykobyletska — APPROVED** [review]

**Pull request merged**